gdx/src/com/badlogic/gdx/Input.java

```diff
@@ -933,8 +933,8 @@ public enum Orientation {
 		Landscape, Portrait
 	}
 
-	/** Only viable on desktop, GWT and Android 8+. Will confine the mouse cursor location to the window and hide the mouse cursor.
-	 * X and y coordinates are still reported as if the mouse was not catched.
+	/** Only viable on the desktop. Will confine the mouse cursor location to the window and hide the mouse cursor. X and y
```

Comment:
It looks like GWT hasn't changed here, but the mention of being viable on GWT was removed anyway. Was it incorrect before?